docs/accessControl.md

```diff
@@ -66,13 +66,13 @@ Every course belongs to an **institution** and by default access is only allowed
-| `Public`    | Access is normally allowed, but is blocked when the student is signed in for an exam in the Computer-Based Testing Facility (CBTF).                                                                                              |
-| `Exam`      | Access is normally blocked, and is only allowed when the student is signed in to the Computer-Based Testing Facility (CBTF). The `examUuid` should also be specified in the same access rule to limit access to a specific exam. |
-| no mode set | Access is allowed anytime, both during CBTF exams and outside of exams.                                                                                                                                                          |
+| Mode        | When is access allowed?                                                                                                                                                                                                                                                               |
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Public`    | Access is normally allowed, but is blocked when the student is checked in on PrairieTest, either in a Computer-Based Testing Facility (CBTF) or in a course-managed session.                                                                                                          |
+| `Exam`      | Access is normally blocked, and is only allowed when the student is checked in on PrairieTest, either in a Computer-Based Testing Facility (CBTF) on in a course-managed session. The `examUuid` should also be specified in the same access rule to limit access to a specific exam. |
+| no mode set | Access is allowed anytime, both during PrairieTest exams and outside of exams.                                                                                                                                                                                                        |
 
-In general usage it is best to set `"mode": "Public"` for any homework (assessments that students should do at home or without special access control), and to set both `"mode": "Exam"` and `"examUuid"` for exams in the Computer-Based Testing Facility (CBTF). This will make it so that exams are only accessible in the CBTF, and homework is _not_ accessible during exams.
+In general usage it is best to set `"mode": "Public"` for any homework (assessments that students should do at home or without special access control), and to set both `"mode": "Exam"` and `"examUuid"` for exams with controlled accessed by PrairieTest. This will make it so that exams are only accessible in the CBTF or a controlled course session, and homework is _not_ accessible during exams.
```

Comment:
```suggestion
In general usage it is best to set `"mode": "Public"` for any homework (assessments that students should do at home or without special access control), and to set both `"mode": "Exam"` and `"examUuid"` for exams with access controlled by PrairieTest. This will make it so that exams are only accessible in the CBTF or a controlled course session, and homework is _not_ accessible during exams.
```